includes/polling/applications/fail2ban.inc.php

```diff
@@ -44,3 +49,43 @@
 
     $int++;
 }
+
+//
+// component processing for fail2ban
+//
+$device_id=$device['device_id'];
```

Comment:
I am a bit lost as to what that will accomplish. That just returns an array for jail names.

Or do you mean modifying it and adding a second function that returns the fail2ban component? That could also be use used there as well.

On another note, what is the proper way to bail out of a poller? The problem I found with this I found that if a host goes unresponsive in the middle of polling, it will update the component to say it has no jails.